pkgs/development/node-packages/composition-v10.nix

```diff
@@ -1,4 +1,4 @@
-# This file has been generated by node2nix 1.7.0. Do not edit!
+# This file has been generated by node2nix 1.6.1. Do not edit!
```

Comment:
It is weird that this was now regenerated using an older version of node2nix. Did you generated it from an up-to-date nixpkgs repository?

Reply:
ah, yes I have older version managed by npm -g, I should ofc install it via nix (but there were some bugs fixed upstream that I needed for something sometime).

Will fix after work